src/chromatix/field.py

```diff
@@ -133,6 +134,15 @@ def dx(self) -> Array:
         """
         return _broadcast_2d_to_grid(self._dx, self.ndim)
 
+    @property
+    def shift_yx(self) -> Array:
+        """
+        The shift in microns of the field in the destination plane. Defined as
```

Comment:
Great except we don't fix the units to microns, we just ask that all the units are consistent. So maybe just say shift?